charts/adot-exporter-for-eks-on-ec2/values.yaml

```diff
@@ -146,7 +151,7 @@ fluentbit:
     readFromTail: "On"
 
 fargateLog:
-  enabled: false
+  enabled: true
```

Comment:
I'm not sure I follow why Fargate logging should be enabled in a context where the data plane is EC2?

Reply:
Fixed. It was for test.